src/literals.py

```diff
@@ -90,6 +90,12 @@ class Status(Enum):
     TLS_NOT_READY = StatusLevel(MaintenanceStatus("Waiting for TLS to be ready"), "DEBUG")
     TLS_PEER_CA_ROTATING = StatusLevel(MaintenanceStatus("Rotating peer CA..."), "DEBUG")
     TLS_CLIENT_CA_ROTATING = StatusLevel(MaintenanceStatus("Rotating client CA..."), "DEBUG")
+    TLS_CERTS_EXPIRING = StatusLevel(
+        MaintenanceStatus(
+            "TLS certificates expiring soon. Please ensure new certificates are provided."
```

Comment:
Could we be slightly more accurate here in specifying the type of the expiring certificate? e.g: `TLS 'client & peer' certificates expiring soon` or `TLS 'client' certificates expiring soon` 

That may require reworking the `check_certificate_validity` method